ingeniamotion/wizard_tests/feedbacks_tests/digital_hall_test.py

```diff
@@ -53,8 +60,10 @@ def halls_extra_settings(self):
             del self.backup_registers[self.axis][self.VELOCITY_FEEDBACK_FILTER_1_FREQUENCY_REGISTER]
 
     @BaseTest.stoppable
-    def suggest_polarity(self, pol):
+    def suggest_polarity(self, pol: Feedbacks.Polarity) -> None:
         polarity_uid = self.FEEDBACK_POLARITY_REGISTER
         pair_poles_uid = self.DIG_HALL_POLE_PAIRS_REGISTER
+        if not isinstance(self.pair_poles, int):
```

Comment:
Same here, check if it is None is more accurate

Reply:
Same as above.